react-frontend/src/profile/ListingCollection.js

```diff
@@ -16,7 +46,7 @@ function ListingCollection(props) {
               <p> There are no {props.title} </p>
             ) : (
             <ul className="collection-list collection-list-listing">
-            {props.bookmarks.map((bookmark) => (
+            {bookmarks.map((bookmark) => (
```

Comment:
Is the listing collection using bookmarks to track its contents? Or is the leftover from making them separate components?

Reply:
Yes this naming is the result of making them separate components, I'll fix the naming here, thanks for pointing that out

Reply:
> Is the listing collection using bookmarks to track its contents? Or is the leftover from making them separate components?

Yes this naming is the result of making them separate components, I'll fix the naming here, thanks for pointing that out